FreeRTOS/FreeRTOS-Plus-TCP
# IPv6/Multi: Reduced complexity FreeRTOS_IP.c

Description
-----------
The module FreeRTOS_IP.c didn't have a high complexity score. Nevertheless I made some changes to make the central function `prvIPTask()` easier to read. I created 6 helper functions:

prvIPTask()
- prvIPTask_Initialise()
- prvIPTask_WaitForEvent()
- prvIPTask_HandleBindEvent()
- prvIPTask_HandleAcceptEvent()
- prvIPTask_HandleSelectEvent()
- prvIPTask_CheckPendingEvents()


Test Steps
-----------
I made the changes a long time ago already, and they have been changed in many situations.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
